src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but in this case that is not the case. Too many blank lines will increase the height of the code, which increases the amount of scrolling. Same for the rest of your code!